allennlp/models/semantic_parsing/wikitables/wikitables_mml_semantic_parser.py

```diff
@@ -93,16 +94,21 @@ def __init__(self,
                                                           action_embedding_dim=action_embedding_dim,
```

Comment:
I missed this earlier, but here it's just `super().__init__(...)`, too.

Reply:
Done.